tests/JustSaying.IntegrationTests/Logging/LogContextTests.cs

```diff
@@ -122,7 +122,7 @@ public async Task HandleMessageFromQueueLogs_ShouldHaveContext(bool handlerShoul
             handleMessage.ShouldNotBeNull();
 
             handleMessage.LogLevel.ShouldBe(level);
-            handleMessage.Exception?.Message.ShouldBe(exceptionMessage);
+            Assert.Equal(exceptionMessage, handleMessage.Exception?.Message);
```

Comment:
```suggestion
            handleMessage.Exception.ShouldNotBeNull();
            handleMessage.Exception.Message.ShouldBe(exceptionMessage);
```

Reply:
This isn't always the case, as its a data driven test, and the exception should only not be null if an exception message is supplied to be thrown. I think we could check to see if the message isnt null and check that way though, to keep things shouldly.

Reply:
Fixed 👍